hyperSpec/vignettes/plotting.Rmd

```diff
@@ -152,22 +152,21 @@ Besides, package **tripack**[`r cite_pkg("tripack")`] is mentioned, but not used
 
 # Preliminary Calculations {-}
 
-For some plots of the `chondro`{.r} dataset, the pre-processed spectra and their cluster averages $\pm$ one standard deviation are more suitable:
+For some plots of the `faux_cell`{.r} dataset, the pre-processed spectra and their cluster averages $\pm$ one standard deviation are more suitable:
 
-```{r preproc-chondro}
-chondro <- hyperSpec::chondro
+```{r preproc-faux-cell}
+set.seed(1)
+faux_cell <- generate_faux_cell()
 
-chondro.preproc <- chondro - spc.fit.poly.below(chondro)
-chondro.preproc <- chondro.preproc / rowMeans(chondro)
-chondro.preproc <- chondro.preproc - quantile(chondro.preproc, 0.05)
+faux_cell_preproc <- faux_cell - spc.fit.poly.below(faux_cell)
+faux_cell_preproc <- faux_cell_preproc / rowMeans(faux_cell)
+faux_cell_preproc <- faux_cell_preproc - quantile(faux_cell_preproc, 0.05)
```

Comment:
We probably don't want this pre-processing step with `faux_cell`

Reply:
Baseline correction is a necessary step.